src/contributing/void-docs/submitting.md

```diff
@@ -0,0 +1,56 @@
+## Submitting Changes
+
+### Requirements
+
+To clone the repository and push changes git is required.
+
+```
+# xbps-install -S git
+```
+
+Building the Void Handbook locally requires
+[mdBook](https://rust-lang-nursery.github.io/mdBook/) to be installed. To get
+it:
+
+```
+# xbps-install -S mdBook
+```
+
+### Forking
+
+To fork the repository a [github account](https://github.com/join) is needed.
+After the account is created follow github's
+[guide](https://help.github.com/en/articles/fork-a-repo) on setting up a fork.
+
+Create a new
+[branch](https://help.github.com/en/articles/creating-and-deleting-branches-within-your-repository)
+to edit, and then clone the repository onto your computer and enter it:
+
+```
+$ git clone https://github.com/YOUR_USERNAME/void-docs.git
+$ cd void-docs
+```
+
+After editing the file(s), commit the changes and push them to the forked
+repository:
+
+```
+$ git add <EDITED_FILE>
+$ git commit -m "<COMMIT_MESSAGE>"
+$ git push
+```
+
+> The commit message should be in the form of `something: reduced section x and
+> moved to it's own page`
+
+Pull requests should only contain a single commit. If a change is made after the
+initial commit, or the commit message needs to be updated the changes will need
+to be squashed. `git add` the changed files and then run `git commit --amend`.
+The updated commit will need to be force pushed: `git push --force`.
+
+If multiple commits are made they'll need to be squashed into one with `git
+rebase -i HEAD~X` where X is the number of commits that need to be squashed. In
+the editor that shows up change all mentions of `pick` to `squash` except for
```

Comment:
I think this topic is a little bigger than this, and can be left out of this guide. Maybe we could just mention that an editor will open to choose which commits to squash, and again to choose the commit message to use. See [git-rebase(1)](https://man.voidlinux.org/git-rebase.1) for more information.

Reply:
Thanks I just don't want to try to *fully* explain how to do that because it's one of the less elegant processes of git ;)